keras/backend/theano_backend.py

```diff
@@ -1008,6 +1008,12 @@ def resize_images(x,
             output._keras_shape[axis_1] *= height_factor
             output._keras_shape[axis_2] *= width_factor
             output._keras_shape = tuple(output._keras_shape)
+    elif interpolation == 'bicubic':
+        output = resize_images(x,
+                               height_factor,
+                               width_factor,
+                               data_format,
+                               interpolation='nearest')
```

Comment:
Can you explain the `interpolation='nearest'` here?

Reply:
@GabrielBianconi Because the optional interpolation argument is set to `nearest`, and the `bicubic` interpolation is not supported in Theano. So when you run `K.resize_images` with the `bicubic` interpolation it will just call the function with the `nearest` interpolation. Little hard to explain but hopefully I cleared that up

Reply:
Wrong Gabriel @abrad1212 @gabrieldemarmiesse

Reply:
@GabrielBianconi Sorry about that. I was writing this in one of my high school classes.